### PR TITLE
Fix Issue #300

### PR DESCRIPTION
**Pull request by Stephan972**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -116,7 +116,7 @@
   </scm>
 
   <properties>
-    <java-build>1.6</java-build>
+    <java-build>1.7</java-build>
```
Comment (typekpb) [inline pom.xml]: this change is not related to issue reported (anyway covered in your other pull request), please revert

**Comment by typekpb** [comment]
btw. from my point of view, this approach is painful, but less than the one we have currently. So I'd go for it.

**Comment by quintonm** [comment]
I agree with @typekpb.  Please revert the change to the compiler settings.  We can merge the rest.

**Comment by typekpb** [comment]
@quintonm @felixbarny well, I guess there is a way to revert the change for @Stephan972 . However I don't have time to play with it right now. How about merging and reverting compiler setting in a next commit? (so that we can polish dependencies in the upcomming version?) 

Volunteers out there? (for proper solution or committing afterwards?)

**Comment by felixbarny** [comment]
I could merge and revert the compiler settings if you want. We could release 2.2.1 then I think. But I don't know what a proper solution would look like.

**Comment by typekpb** [comment]
Feel free to do so, please
